### PR TITLE
Update httpclient

### PR DESCRIPTION
**Pull request by piotrekfus91**

### CODE REVIEW

File: mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerIntegrationTest.groovy [hunk 1]
```diff
@@ -14,6 +14,7 @@ import org.apache.http.entity.ContentType
 import org.apache.http.entity.StringEntity
 import org.apache.http.impl.client.CloseableHttpClient
 import org.apache.http.impl.client.HttpClients
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager
```
Comment (alien11689) [inline mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerIntegrationTest.groovy]: why do we need it?

File: pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <maven-compiler-plugin.version>3.1</maven-compiler-plugin.version>
         <groovy.version>2.5.7</groovy.version>
-        <httpclient.version>4.3.6</httpclient.version>
+        <httpclient.version>4.4</httpclient.version>
```
Comment (alien11689) [inline pom.xml]: why do not upgrade to version 4.5.x as dependabot suggests? https://github.com/TouK/http-mock-server/pull/9

**Review by alien11689 — CHANGES_REQUESTED** [review]

File: mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerIntegrationTest.groovy (hunk 1 above)
Comment (piotrekfus91) [reply mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerIntegrationTest.groovy]: Forgot to remove this line :)

File: pom.xml (hunk 2 above)
Comment (piotrekfus91) [reply pom.xml]: Right, I upgraded to the version that caused error.

**Review by alien11689 — APPROVED** [review]

**Pull request merged**